indico/modules/events/templates/management/_lists.html

```diff
@@ -1,5 +1,5 @@
-{% macro render_displayed_entries_fragment(displayed_num, total_num) -%}
-    <span id="filtering-state" class="i-button label icon-list"
+{% macro render_displayed_entries_fragment(displayed_num, total_num , id=None) -%}
+    <span id="{{ 'filtering-state' if not id else id }}" class="i-button label icon-list"
```

Comment:
just `{{ id }}` with the change above. also, just fyi, you could do `{{ id or 'filtering-state' }}` in a case like this